source/documentation/make-updates-at-tax-year-end.html.md

```diff
@@ -591,6 +591,30 @@ You should use the same business and non-business income APIs to allow customers
 
 Before an amendment submission is accepted, the customer must have completed their final declaration for the relevant tax year through software, and the amendment request must be submitted within the designated amendment window. Customers will first need to submit an **Intent to Amend Calculation**, during which additional validation checks will be carried out. If any errors are identified these will be sent to the software, which must display them to the customer or their authorised agent.
 
+<a href="figures/PFA-customer-journey.svg" target="blank"><img src="figures/PFA-customer-journey.svg" alt="Diagram of post-finalisation amendment customer journey" style="width:720px;" /></a>
+<a href="figures/PFA-customer-journey.svg" target="blank">Open the post-finalisation amendment customer journey diagram in a new tab</a>.
+
+1. User starts Post Finalisation Process in Software.
+2. (Optional) Software checks if customer is within the amendment window by calling the List endpoint within the Individual Calculation API.
```

Comment:
This is missing `s` in Calculation. Please update to `Individual Calculations API`, thanks